### ISSUE TITLE
How to subset - filter data for different species

### ISSUE THREAD

**Issue by ericward-noaa** [fate-spatialindicators/consonants#1]
Should we filter surveys to the range that species are observed?

**Comment by jordanwatson** [comment]
If you don't, then wouldn't zeroes in the data bias results on the boundaries?

**Comment by ericward-noaa** [comment]
Right now the way that we're doing it is restricting the estimation to the spatial domain that contains positive tows for each species. There's probably other ways to do this, but wanted to flag it as a topic for discussion / input

**Comment by jordanwatson** [comment]
I guess that by leaving the extended range in there, you provide the opportunity for the species distribution to shift, which of course is the whole question....

**Comment by Lewis-Barnett-NOAA** [comment]
Yes, Eric and I discussed trying a more conservative threshold for defining the spatial domain in some follow-up work.  Something like the convex hull that encompasses 95% of the positive observations?

**Comment by ericward-noaa** [comment]
Our subgroup was worried about the potential to have estimates of environmental effects be biased with all the zeros. For example, if a species was only observed in the northern half of the Cal Current, and the temp gradient means it's warmer in the south, fitting a model to the full dataset would overestimate the temp effect on ranges.
